config/initializers/headers.rb

```diff
@@ -0,0 +1,13 @@
+headers = [
+  'HTTP/1.1 200 OK',
+  'Content-Type: text/event-stream',
+  'Cache-Control: no-cache, no-store',
+  'Connection: close'
+]
+
+origin = SseRailsEngine.access_control_allow_origin
```

Comment:
why bother assigning this to a variable? Line lengths are generally >100 cols these days.